### PR TITLE
Move Visualization server and Viewer CRD to an overlay

### PR DESCRIPTION
**Pull request by rimolive**
- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [X] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [X] JIRA link(s): https://issues.redhat.com/browse/RHODS-6515
- [X] The Jira story is acked
- [ ] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

### CODE REVIEW

**Review by DharmitD — COMMENTED** [review]
/lgtm

**Review by HumairAK — APPROVED** [review]

**Review by anishasthana — APPROVED** [review]
/lgtm

File: data-science-pipelines/overlays/component-visualization-server/deployments/ds-pipeline-visualizationserver.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   annotations:
     alpha.image.policy.openshift.io/resolve-names: 'visualization-server:1.3.1'
     image.openshift.io/triggers: >-
-      [{"from":{"kind":"ImageStreamTag","name":"visualization-server:1.3.1"},"fieldPath":"spec.template.spec.containers[?(@.name==\"ds-pipeline-visualizationserver\")].image"}]
+      [{"from":{"kind":"ImageStreamTag","name":"visualization-server:1.3.1"},"fieldPath":"spec.template.spec.containers[?(@.name==\"ml-pipeline-visualizationserver\")].image"}]
```
Comment (LaVLaS) [inline data-science-pipelines/overlays/component-visualization-server/deployments/ds-pipeline-visualizationserver.yaml:7]: @rimolive Should this container name stay `ds-pipeline-visualizationserver`?

Comment (rimolive) [reply data-science-pipelines/overlays/component-visualization-server/deployments/ds-pipeline-visualizationserver.yaml:7]: There were no code changes, just files moved from one location to another. This shouldn't be an issue as these manifests were tested before.

Comment (LaVLaS) [reply data-science-pipelines/overlays/component-visualization-server/deployments/ds-pipeline-visualizationserver.yaml:7]: Gotcha.

**Pull request merged**